docs/configuration/radio/device.mdx

```diff
@@ -11,24 +11,33 @@ The device config options are: Role, Serial Output, and Debug Log. Device config
+| LOST_AND_FOUND | Broadcasts location as message to default channel regularly for recovery efforts.                                                              | Used for recovery efforts of a lost device.                                                                                            |
+| SENSOR         | Broadcasts priority telemetry packets, optimized for sensor data collection.                                                                   | Deploying in scenarios where gathering environmental or other sensor data is crucial, with efficient power usage and frequent updates. |
+| TAK            | Optimized for ATAK system communication, reduces routine broadcasts.                                                                           | Integration with ATAK systems for communication in tactical or coordinated operations.                                                 |
+| REPEATER       | Dedicated device that focuses solely on extending mesh network coverage by relaying messages with minimal overhead. **Not visible** in Nodes list. | Best positioned in strategic locations to maximize the network's overall coverage. Device **is not shown** in topology.                    |
+| ROUTER         | Dedicated device that focuses solely on extending mesh network coverage by relaying messages. **Visible** in Nodes list.                           | Best positioned in strategic locations to maximize the network's overall coverage. Device **is shown** in topology.                        |
+| ROUTER_CLIENT  | Combination of both ROUTER and CLIENT.                                                                                                         | Devices in a strategic position for priority routing that need to also serve as a standard CLIENT.                                     |
+
+#### Role Comparison
+| Device Role    | Bluetooth/Wi-Fi Enabled | Screen Enabled | Power Consumption | Retransmit | Prioritized Routing | Visible in Nodes List |
+| -------------- | ----------------------- | -------------- | ----------------- | ---------- | ------------------- | --------------------- |
+| CLIENT         | Yes                     | Yes            | Regular           | Yes        | No                  | Yes                   |
+| CLIENT_MUTE    | Yes                     | Yes            | Lowest            | No         | No                  | Yes                   |
+| CLIENT_HIDDEN  | Yes                     | Yes            | Lowest            | Local only | No                  | No                    |
+| TRACKER        | Optional                | No             | Regular           | No         | No                  | Yes                   |
+| LOST_AND_FOUND | Optional                | No             | Regular           | No         | No                  | Yes                   |
+| SENSOR         | Optional                | No             | High              | No         | No                  | Yes                   |
```

Comment:
Why Optional for these three?

Reply:
Honestly can't remember why. I don't like it anymore. I think we just want whatever the default is, which I actually do not know what that is.